react-toggle-switch/my-app/src/ToggleSwitch.tsx

```diff
@@ -2,20 +2,23 @@ import { useState } from 'react';
 import './App.css';
 
 export function ToggleSwitch() {
-  const [toggle, setToggle] = useState('off');
-
+  const [toggle, setToggle] = useState(true);
+  let onOff = '';
+  if (toggle) {
+    onOff = 'on';
+  } else {
+    onOff = 'off';
+  }
   return (
-    <>
-      <div className="container">
-        <div className={toggle}>
-          <div
-            onClick={() => {
-              toggle === 'off' ? setToggle('on') : setToggle('off');
-            }}
-            className="switch"></div>
-        </div>
-        <div className="on-off-text">{toggle}</div>
+    <div className="container">
+      <div className={onOff}>
+        <div
+          onClick={() => {
+            toggle === true ? setToggle(false) : setToggle(true);
+          }}
+          className="switch"></div>
       </div>
-    </>
+      <div className="on-off-text">{onOff}</div>
```

Comment:
You are using the value of `onOff` both to determine the class name and to display text to the user. You should instead do something similar to line 17 and use `toggle` to calculate the text to display.